stable/redis/values.yaml

```diff
@@ -356,6 +357,7 @@ slave:
     annotations: {}
     labels: {}
     loadBalancerIP:
+    # loadBalancerSourceRanges: ["10.0.0.0/8"]
```

Comment:
Please update the **values-production.yaml** too

Reply:
Updated in 87e63e0805590f197e9d3458dffbaf3ce830c363